service/pkg/server/services.go

```diff
@@ -136,9 +139,19 @@ func startServices(ctx context.Context, cfg config.Config, otdf *server.OpenTDFS
 				}
 			}
 
+			// If the service provides its own logging config, create a new logger for that service using that config
+			if extractedLogger, err := tryExtractingServiceLoggerConfig(cfg.Services[svc.Namespace]); err == nil {
+				slog.Debug("configuring logger")
+				newSvcLogger, err := logging.NewLogger(extractedLogger)
+				if err != nil {
+					return fmt.Errorf("could not start logger: %w", err)
+				}
+				thisSvcLogger = newSvcLogger.With("namespace", ns)
```

Comment:
My understanding was that we'd want a `service` field that was the svc.Namespace value for all services, not just ones that have a custom configuration

Reply:
theres a line above that mimics this for the default service logger https://github.com/opentdf/platform/blob/8240645270e1cd25001c6f40d4276093c6e3cb23/service/pkg/server/services.go#L123
the line here just does the same but for the new service specific logger

Reply:
I think this logic might need to be defined outside the service loop because for example `policy` contains N number of services.

- attributes
- namespaces,
- subjectmappings
- ...